src/auth.rs

```diff
@@ -103,29 +107,14 @@ impl AuthStore {
         self.file.seek(SeekFrom::End(0)).unwrap();
         write!(self.file, "{}\n", auth).unwrap();
     }
-    pub fn remove(&mut self, i: usize) {
-        // remove in the vec
-        self.auths.remove(i);
-        // remove in the file
-        let mut reader = BufReader::new(&self.file);
-        reader.seek(SeekFrom::Start(0)).unwrap();
-
-        let mut lines = reader.lines().collect::<Vec<_>>();
-        lines.remove(i).unwrap();
 
-        // Prefferably I should just be writing to a new file, and then
-        // just swap the files when I'm finished writing, but realisticly
-        // there is no point in this type of redundncy at this point in the
-        // project.
+    pub fn remove(&mut self) {
```

Comment:
At this point, this function should probably be renamed to `file_sync`, and I'm not sure it should stay public.